lib/resque_bus/subscriber.rb

```diff
@@ -0,0 +1,28 @@
+module QueueBus
```

Comment:
That's reasonable.  I think this is correct-by-construction, but the extra `require` won't harm anything, and if it wasn't loaded for whatever, things just explode.